src/api/spec/features/webui/requests_spec.rb

```diff
@@ -83,6 +83,8 @@
 
     describe 'for packages' do
       it 'can be submitted' do
+        skip_if_bootstrap
+
```

Comment:
:thinking: are they really that different? What's the cause?

Reply:
@DavidKang I had one test failure because capybara was using a very small screen size. Try out e2ecb0eaf4e52f58f5998dea9f494011038ba382. It might solve some of the test failures.